core/preference/preference.go

```diff
@@ -26,17 +26,49 @@ const (
 	TraitInputNumber      TraitInput = "number"
 )
 
+const (
+	// platform default traits
+	PlatformDisableOrgsOnCreate = "disable_orgs_on_create"
+	PlatformDisableOrgsListing  = "disable_orgs_listing"
+	PlatformDisableUsersListing = "disable_users_listing"
+	PlatformInviteWithRoles     = "invite_with_roles"
+	PlatformInviteMailSubject   = "invite_mail_template_subject"
+	PlatformInviteMailBody      = "invite_mail_template_body"
+	PlatformMailOTPBody         = "oidc_mail_otp_body"
+	PlatformMailOTPSubject      = "oidc_mail_otp_subject"
+	PlatformMailOTPValidity     = "oidc_mail_otp_validity"
+	PlatformMailLinkSubject     = "mail_link_subject"
+	PlatformMailLinkBody        = "mail_link_body"
+	PlatformMailLinkValidity    = "mail_link_validity"
```

Comment:
Hey! I just realized we have moved even authentication configs, we decided earlier to not move them as oidc configs are still part of it. As they are secret, we need to encrypt them and currently preferences don't support encrypting credentials. So keeping half config here and half there doesn't make sense.